### DISCUSSION TITLE
ImageClip(imagePath).set_duration(lengthDuretion)   Help on lengthDuretion

### DISCUSSION THREAD

**Discussion by bongblender** [Zulko/moviepy#1720] [Q&A]
ImageClip(imagePath).set_duration(lengthDuretion)

I am a little bit noob, so I am here for help. 
How can I set ImagePath and lengthDuretion at once ... from 2 different arrays. 
basically, for each different photo, I want a different time span.
now, what I am doing wrong, or what should I do.?

for i in imageFileList:
_______clip = [ImageClip(i).set_duration(I want to set this also)]
concat_clip = concatenate_videoclips(clips, method="compose")
concat_clip.write_videofile("test.mp4", fps=24)

Thanks in advance...

**Comment by keikoro** [comment]
Please use the regular issue tracker or ask a question on SO, the discussion board isn't really the best place to post these kinds of questions.
